src/sortStudents.ts

```diff
@@ -1,7 +1,76 @@
-// describe Student type
-// create and export SortType enum
-// create SortOrder type
+type Student = {
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number [];
+};
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+export enum SortType {
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+type SortOrder = 'asc' | 'desc';
+
+function AverageMark(student: Student): number {
```

Comment:
it's a function = verb

```suggestion
function calcAverageMark(student: Student): number {
```